static/app/views/releases/detail/overview/index.tsx

```diff
@@ -458,7 +458,7 @@ class ReleaseOverview extends AsyncView<Props> {
                           )}
                           <ReleaseDetailsPageFilters>
                             <EnvironmentPageFilter />
-                            <StyledPageTimeRangeSelector
+                            <PageTimeRangeSelector
```

Comment:
`PageTimeRangeSelector` now matches button/form input heights, so there's no need to re-style it.
<img width="862" alt="Crash Free Session F" src="https://user-images.githubusercontent.com/44172267/196279105-ea660fb6-8461-4616-826f-d34d5dfc9425.png">